src/filters/filter_cue.cpp

```diff
@@ -379,6 +417,7 @@ namespace dmxfish::filters {
                 start_new_cue();
                 cue_end_handling_real = cues.at(active_cue).end_handling;
             }
+            update_parameter_gui();
```

Comment:
I think we should update this only every x cycles. How about every 200 (e.g. = 5sec)?

Reply:
her the function is only called if the time exceed the actual frame. So it should probably not that often.
But if you would prefer to update it every 200 cycles (instead of after every frame), i could do that as well.

Reply:
Nah, updating it once every frame change should be sufficient.